script/rest/update-files.js

```diff
@@ -45,15 +46,16 @@ async function main() {
   // When the input parameter type is decorate-only, use the
   // `github/docs-internal` repo to generate a list of schema files.
   // Otherwise, use the `github/github` list of config files
-  const schemas = decorateOnly ? await readdir(DEREFERENCED_DIR) : await getSchemas()
+  const schemas = decorateOnly
+    ? await readdir(DOCS_DEREF_OPENAPI_DIR)
+    : await getSchemas(OPEN_API_RELEASES_DIR, includeDeprecated, includeUnpublished, versionsArray)
 
   // Generate the dereferenced OpenAPI schema files
   if (!decorateOnly) {
     await getBundledFiles(schemas)
   }
   // Decorate the dereferenced files in a format ingestible by docs.github.com
   await decorate(schemas)
-
```

Comment:
>  console.log(
     '\n🏁 The static REST API files are now up-to-date with your local `github/github` checkout. To revert uncommitted changes, run `git checkout lib/rest/static/*`.\n\n'
   )
}
async function getBundledFiles(schemas) {
  // Get the github/github repo branch name and pull latest
  const githubBranch = execSync('git rev-parse --abbrev-ref HEAD', { cwd: GITHUB_REP_DIR })
    .toString()
    .trim()
  // Only pull master branch because development mode branches are assumed
  // to be up-to-date during active work.
  if (githubBranch === 'master') {
    execSync('git pull', { cwd: GITHUB_REP_DIR })
  }
  // Create a tmp directory to store schema files generated from github/github
  rimraf.sync(TEMP_DOCS_DIR)
  await mkdirp(TEMP_DOCS_DIR)
  console.log(
    `\n🏃‍♀️🏃🏃‍♀️Running \`bin/openapi bundle\` in branch '${githubBranch}' of your github/github checkout to generate the dereferenced OpenAPI schema files.\n`
  )
  // Format the command supplied to the bundle script in `github/github`
  const bundlerOptions = await getBundlerOptions()
  try {
    console.log(`bundle -o ${TEMP_DOCS_DIR} ${bundlerOptions}`)
    execSync(
      `${path.join(GITHUB_REP_DIR, 'bin/openapi')} bundle -o ${TEMP_DOCS_DIR} ${bundlerOptions}`,
      { stdio: 'inherit' }
    )
  } catch (error) {
    console.error(error)
    const errorMsg =
      '🛑 Whoops! It looks like the `bin/openapi bundle` command failed to run in your `github/github` repository checkout.\n\n✅ Troubleshooting:\n - Make sure you have a codespace with a checkout of `github/github` at the same level as your `github/docs-internal` repo before running this script. See this documentation for details: https://thehub.github.com/epd/engineering/products-and-services/public-apis/rest/openapi/openapi-in-the-docs/#previewing-changes-in-the-docs.\n - Ensure that your OpenAPI schema YAML is formatted correctly. A CI test runs on your `github/github` PR that flags malformed YAML. You can check the PR diff view for comments left by the OpenAPI CI test to find and fix any formatting errors.\n\n'
    throw new Error(errorMsg)
   }

   execSync(